deploy/README.md

```diff
@@ -1,4 +1,4 @@
-# Deployment Guide
+# Deployment Guide for release v2.4
```

Comment:
Not sure if that's a good idea, this will get old very quick.

Reply:
Yes, that's the question what to put here in the main branch.
Maybe it should be `for unreleased version`?

This particular change here was to show how I think we should keep things in release branches.

Reply:
Yes, that's the question what to put here in the main branch.
Maybe it should be `for unreleased version`?

This particular change here was to show how I think we should keep things in release branches.

Reply:
Changed.